### DISCUSSION TITLE
CCM not labelling nodes

### DISCUSSION THREAD

**Discussion by Th4n05** [sergelogvinov/proxmox-cloud-controller-manager#101] [Q&A]
Hi @sergelogvinov, thanks a lot for this project.
I'm trying to use CCM, I've already configured kubelet cloud provider and alpha.kubernetes.io/provided-node-ip annotation.
The execution does not go beyond the Instance metadata configuration.
<img width="1397" alt="Screenshot 2024-02-19 at 18 30 59" src="https://github.com/sergelogvinov/proxmox-cloud-controller-manager/assets/43700726/7ee3f806-7f54-44c3-8f4d-0ff4af75d895">
I'm on kubernetes 1.27.5 using calico and coredns.
At the end, the controller does not labels any node, do you have any advice to understand what the problem could be.

**Comment by sergelogvinov** [comment]
Hello,

Only kubelet should sets `alpha.kubernetes.io/provided-node-ip`, 
1. you need to run `kubelet` with params - `kubelet --cloud-provider=external --node-ip=1.2.3.4 --hostname-override=worker-05`
2. the node name == proxmox vm name
3. before restart kubelet - delete the node resource - `kubectl delete node worker-05`


I hope it will help you...

**Comment by Th4n05** [comment]
Thank you very much  @sergelogvinov, deleting the resource and restarting it worked well.
